packages/esm-patient-tests-app/src/test-orders/add-test-order/test-order-form.component.tsx

```diff
@@ -118,7 +118,7 @@ export function LabOrderForm({
   const { orderReasons } = useOrderReasons(orderReasonUuids);
 
   const filterItemsByName = useCallback((menu) => {
-    return menu?.item?.value?.toLowerCase().includes(menu?.inputValue?.toLowerCase());
+    return menu?.item?.display?.toLowerCase().includes(menu?.inputValue?.toLowerCase());
```

Comment:
Shouldn't we have a check that the string matching should only happen when the inputValue is a valid and non-empty string?

Reply:
Thanks @vasharma05. Updated